src/DotNetty.Common/ThreadDeathWatcher.cs

```diff
@@ -59,7 +59,10 @@ static void Schedule(Thread thread, Action task, bool isWatch)
 
             if (Interlocked.CompareExchange(ref started, 1, 0) == 0)
             {
-                var watcherThread = new Thread(s => ((IRunnable)s).Run());
+                var watcherThread = new Thread(s => ((IRunnable)s).Run())
+                {
```

Comment:
nit: `watcherThread.IsBackground = true;` would be easier on the eyes.

Reply:
fixed